Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -905,24 +904,38 @@ public static void HandleUnlock(Session session, params string[] parameters)
         }
 
         // gamecast <message>
-        [CommandHandler("gamecast", AccessLevel.Envoy, CommandHandlerFlag.RequiresWorld, 1)]
+        [CommandHandler("gamecast", AccessLevel.Envoy, CommandHandlerFlag.RequiresWorld, 1,
+            "Sends a world-wide broadcast.",
+            "<message>\n" +
+            "This command sends a world-wide broadcast to everyone in the game. Text is prefixed with 'Broadcast from (admin-name)> '.\n" +
+            "See Also: @gamecast, @gamecastemote, @gamecastlocal, @gamecastlocalemote.")]
         public static void HandleGamecast(Session session, params string[] parameters)
         {
             // > Broadcast from usage: @gamecast<message>
             // This command sends a world-wide broadcast to everyone in the game. Text is prefixed with 'Broadcast from (admin-name)> '.
             // See Also: @gamecast, @gamecastemote, @gamecastlocal, @gamecastlocalemote.
             // @gamecast - Sends a world-wide broadcast.
 
-            // TODO: output
+            session.Player.HandleActionWorldBroadcast($"Broadcast from {session.Player.Name}> {string.Join(" ", parameters)}", ChatMessageType.WorldBroadcast);
         }
 
         // add <spell>
-        [CommandHandler("add", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 1)]
+        [CommandHandler("add", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 1,
+            "Adds the specified spell to your own spellbook.",
+            "<spellid>")]
         public static void HandleAdd(Session session, params string[] parameters)
```

Comment:
I don't like the idea of deviating from known commands in AdminCommands. I'll add an addspell command in DebugCommands which calls this function.